create-node-api/Makefile

```diff
@@ -7,7 +7,4 @@ FILES = # add the file that need to be package here like $(NAME)/create-node-api
 SCRIPTS = postinst
 
 include ../makefiles/common-scala-plugin.mk
-
-target/$(NAME)/create-node-api-schema.sql:
-	cp ./src/main/resources/create-node-api-schema.sql target/$(NAME)/
-
+println
```

Comment:
what does println do ? you need at least a target/command so the makefile is valid  ?

Reply:
Apart that comment it can be merged !!